library/dash/src/main/java/com/google/android/exoplayer2/source/dash/DashMediaSource.java

```diff
@@ -983,4 +994,13 @@ public Long parse(Uri uri, InputStream inputStream) throws IOException {
 
   }
 
+  private static final class HttpHeadTimeParser implements ParsingLoadable.Parser<Long> {
+
+    @Override
+    public Long parse(Uri uri, InputStream inputStream) throws IOException {
+      return new URL(uri.toString()).openConnection().getDate();
```

Comment:
We could do with finding a way for this request to go through `dataSource`. If an application has opted to use `OkHttp` then we don't really want this one connection going through a different stack.